templates/kubernetes/overlays/production/kustomization.yml

```diff
@@ -19,3 +19,4 @@ configMapGenerator:
     - ENVIRONMENT=production
     - DOMAIN=<% index .Params `productionHostRoot` %>
     - S3_BUCKET=files.<% index .Params `productionHostRoot` %>
+    - GRAPHQL_BASE_URL=https://<% index .Params `productionBackendSubdomain` %><% index .Params `productionHostRoot` %>
```

Comment:
Please add a template block around these so they are not included if the user chose REST

Reply:
This env variable isn't needed since all APIs wrappers get removed.